docker-compose.yml

```diff
@@ -19,3 +19,6 @@ services:
     # The image chosen here should be kept in sync with the .travis.yml mongo version and with
     # the value in mongomock/__init__.py
     image: mongo:5.0.5
+    ports:
+      - 27017:27017
```

Comment:
This poses a security threat: you're opening a port on the host. If you want to check locally, I'd recommend using
```yml
ports:
  - localhost:27017:27017
```

Reply:
`localhost` doesn't work, but I can use `127.0.0.1` instead. I'm not sure about the attack vector however